src/actions/generate-nft-images.ts

```diff
@@ -0,0 +1,105 @@
+import { Op, Sequelize } from "sequelize";
+
+import db from "src/db";
+
+import { generateNftImage } from "src/modules/generate-bounty-cards";
+
+import { getChainsRegistryAndNetworks } from "../utils/block-process";
+import { isIpfsEnvs } from "src/utils/ipfs-envs-verify";
+import logger from "src/utils/logger-handler";
+
+import ipfsService from "src/services/ipfs-service";
+
+export const name = "generate-nft-images";
+export const schedule = "*/10 * * * *";
+export const description = "Generate NFT image for open bounties";
+export const author = "vhcsilva";
+
+export async function action(issueId?: string) {
+  const bountiesProcessed: any[] = [];
+
+  if (!isIpfsEnvs) {
+    logger.warn(`${name} Missing id, secret or baseURL, for IPFService`);
+    return bountiesProcessed;
+  }
+
+  const entries = await getChainsRegistryAndNetworks();
+  for (const [, {chainId: chain_id,}] of entries) {
+    try {
+      logger.info(`${name} start`);
+
+      const dbEvent =
+        await db.chain_events
+          .findOrCreate({where: {name, chain_id}, defaults: {name, chain_id}})
+          .then(([event,]) => event);
+
+      const where = {
+        chain_id,
+        state: {
+          [Op.notIn]: ["draft", "canceled", "pending"]
+        },
+        fundingAmount: {
+          [Op.eq]: Sequelize.col("issues.fundedAmount")
+        },
+        nftImage: {
+          [Op.eq]: null
+        }
+      }
+
+      const include = [
+        {association: "developers"},
+        {association: "merge_proposals"},
+        {association: "pull_requests"},
+        {association: "network"},
+        {association: "repository"},
+        {association: "transactionalToken"},
+      ];
+
+      const bounties = await db.issues.findAll({
+        where,
+        include,
+        limit: 5
```

Comment:
I think it would be interesting to put the `limit` variable in a constant to improve the understanding and maintenance of the code